src/isDefined.ts

```diff
@@ -13,3 +13,9 @@
 export function isDefined<T>(data: T): data is NonNullable<T> {
   return typeof data !== 'undefined' && data !== null;
 }
+
+export namespace isDefined {
+  export function strict<T>(data: T | undefined): data is T {
```

Comment:
we'd also need docs here

Reply:
do we document variants? I couldn't find an example where we do, the documentation usually resides on the main object

Reply:
ah you're right, it's enough if we have an example 👍